.gitattributes

```diff
@@ -0,0 +1,3 @@
+* text=auto
+.gitattributes text eol=lf
+.gitignore text eol=lf
```

Comment:
should be sufficient without these two lines. Or am I missing something?

Reply:
Why do you think so? `* text=auto` -> *Git will handle the files in whatever way it thinks is best. This is a good default option.*
So the best option isn't always `eol=lf` and so we need line 2.
I don't really get what the difference between line 2 and line 3 is and why you need `.gitattributes` and ´.gitignore´ @Zeroks77 ?